README.md

```diff
@@ -74,15 +74,15 @@ apply plugin: 'app.cash.licensee'
 ```
 
 <details>
-<summary>Snapshots of the development version are available in Sonatype's snapshots repository.</summary>
+<summary>Snapshots of the development version are available in Central Portal Snapshots repository.</summary>
 <p>
 
 ```groovy
 buildscript {
   repositories {
     mavenCentral()
     maven {
-      url 'https://oss.sonatype.org/content/repositories/snapshots/'
+      url 'https://central.sonatype.com/repository/maven-snapshots/'
```

Comment:
Specifying the portal automatically targets this too?

Reply:
I tested something in https://github.com/GradleUp/shadow/pull/1414:

- Versions with `-SNAPSHOT` are automatically published to https://central.sonatype.com/repository/maven-snapshots/
- The version without `-SNAPSHOT` is published to https://repo1.maven.org/maven2/ (https://central.sonatype.com)
- https://oss.sonatype.org/content/repositories/snapshots/ will not sync with https://central.sonatype.com/repository/maven-snapshots/ after the migration.